### PR TITLE
feat: replace green confirm button with red

### PR DESCRIPTION
**Pull request by shivadharmi**
This pull request replaces the green confirm button with red color

Closes https://github.com/helpwave/web/issues/245

**Testing instructions:**
- Go to the organizations' page and click the edit icon of any organization from the list to see organization info.
- Under the Danger Zone section click on the Delete Organization button you will see a confirmation dialogue and it should contain a confirmation button in red color instead of green.

**Additional notes:**
- Refactored the confirmation dialogue to support different types of confirmations. The default confirmation dialogue is Positive Action.

### CODE REVIEW

**Review by florian-sabonchi — APPROVED** [review]

**Comment by florian-sabonchi** [comment]
Good work looks good 🚀

**Comment by shivadharmi** [comment]
Thank you @florian-sabonchi.

**Comment by florian-sabonchi** [comment]
However, we should wait with the merge on more until the others have looked over it

File: tasks/components/modals/ConfirmDialog.tsx [hunk 1]
```diff
@@ -30,7 +32,8 @@ type ConfirmDialogProps = ModalProps & {
   requireAnswer?: boolean,
   onCancel?: () => void,
   onConfirm: () => void,
-  onDecline?: () => void
+  onDecline?: () => void,
+  type: ConfirmDialogType
```
Comment (florian-sabonchi) [inline tasks/components/modals/ConfirmDialog.tsx:36]: I think the name confirmType is more appropriate

Comment (florian-sabonchi) [reply tasks/components/modals/ConfirmDialog.tsx:36]: Excuse me, but I still noticed one small thing. How do you see the  @DasProffi

Comment (MaxSchaefer) [reply tasks/components/modals/ConfirmDialog.tsx:36]: Please use the following instead
```suggestion
  confirmType: ConfirmDialogType
```

Comment (shivadharmi) [reply tasks/components/modals/ConfirmDialog.tsx]: Hey @florian-sabonchi and @MaxSchaefer. I have pushed a new commit with changes. Please take a look at it.

**Comment by florian-sabonchi** [comment]
Seems like that the Pipeline failed aftert that is fixed de can merge

**Review by MaxSchaefer — APPROVED** [review]

**Pull request merged**